lib/view_model/contact_list/contact_list_view_model.dart

```diff
@@ -8,20 +8,35 @@ import 'package:cake_wallet/entities/contact_record.dart';
 import 'package:cake_wallet/entities/contact.dart';
 import 'package:cake_wallet/utils/mobx.dart';
 import 'package:cw_core/crypto_currency.dart';
+import 'package:collection/collection.dart';
 
 part 'contact_list_view_model.g.dart';
 
 class ContactListViewModel = ContactListViewModelBase with _$ContactListViewModel;
 
 abstract class ContactListViewModelBase with Store {
-  ContactListViewModelBase(this.contactSource, this.walletInfoSource, this._currency)
+  ContactListViewModelBase(
+      this.contactSource, this.walletInfoSource, this._currency, this.isAutoGenerateEnabled)
       : contacts = ObservableList<ContactRecord>(),
         walletContacts = [] {
     walletInfoSource.values.forEach((info) {
-      if (info.addresses?.isNotEmpty ?? false) {
-        info.addresses?.forEach((address, label) {
-          final name = label.isNotEmpty ? info.name + ' ($label)' : info.name;
+      if (info.addresses == null || info.addresses!.isEmpty) return;
```

Comment:
`return` will exit the function, but we need it to complete the loop normally and get other contacts